libshvbroker/src/brokerappnode.cpp

```diff
@@ -146,8 +147,7 @@ chainpack::RpcValue BrokerAppNode::callMethodRq(const chainpack::RpcRequest &rq)
 				}
 				emitSendRpcMessage(resp);
```

Comment:
proc je tu ten emit?

Reply:
Aby se poslala odpoved puvodnimu zadateli. Je to asynchronni request, synchronne se vraci `return cp::RpcValue()` a kdyz prijde odpoved z master brokeru tak se posle "opravdova" odpoved.
Takhle jsi mi to ukazal nekdy na samotnem zacatku, pouzivam to od te doby v tscagentovi, nevim jestli se to ted nedela nejak jinak.

Reply:
jasny, dej tam `rootNode()->emitSendRpcMessage(resp);` , at se takhle blbe priste neptam.

Reply:
Opraveno